File: Lecture 002 Recursion Phod Denge/Tail.cpp
#include<iostream> 
using namespace std;

void func(int n){
    //Base Case
    if(n==0) 
        return ;
    
    //Tumhara kaam 
    cout<<n<<endl;
    
    //Bharosa(trust)
    func(n-1);
}

int main(){
    int n;
    cin>>n;
    func(n);
}

File: Lecture 002 Recursion Phod Denge/Tail.java
import java.util.*;
 
public class Tail {
 

static void func(int n)
{
 //Base Case
 if(n==0) 
    return ;
 
 //Tumhara kaam
 System.out.println(n);

 //Bharosa(trust)
 func(n-1);
}
 
// Driver code
public static void main(String[] args)
{
    int n;
    Scanner sc=new Scanner(System.in);  
    n= sc.nextInt();
 
    func(n);
}
}